src/scripts/spot.js

```diff
@@ -358,20 +403,20 @@ module.exports = function (robot) {
     Support = require('./support/spotifySupport')(robot, URL, Queue);
     Assoc = require('./support/spotifyAssoc')(robot);
 
-    //if (process.env.HUBOT_SPOTIFY_PLAYLIST_FILE) {
-    //    // Set up default queue
-    //    setupDefaultQueue(playlistQueue);
-    //
-    //    // Set the default queue on the queue master
-    //    queueMaster.setDefault(playlistQueue);
-    //
-    //    // Add the user queue
-    //    queueMaster.addQueue(Queue);
-    //
-    //    // Conduct the queues (the default queue will
-    //    // play if user queue is empty)
-    //   queueMaster.conduct();
-    //}
+    if (process.env.HUBOT_SPOTIFY_PLAYLIST_ID) {
+        // Set up default queue
+        setupDefaultQueue(playlistQueue);
+
+        // Set the default queue on the queue master
+        queueMaster.setDefault(playlistQueue);
+
+        // Add the user queue
+        queueMaster.addQueue(Queue);
+
+        // Conduct the queues (the default queue will
+        // play if user queue is empty)
+       queueMaster.conduct();
```

Comment:
@andromedado Testing seems to indicate that this is not the case (restarts do not kill the currently playing song). If this is the only concern then I'd strongly recommend merging now and dealing with any issues as they arise, since the current queue implementation on Dibsy "prod" is extremely clunky and annoying